exercises/practice/two-fer/src/test/kotlin/TwoFerTest.kt

```diff
@@ -10,21 +10,18 @@ class TwoFerTest {
     }
 
     @Test
-    @Ignore
     fun aNameGiven() {
         assertEquals("One for Alice, one for me.", twofer("Alice"))
     }
 
     @Test
-    @Ignore
     fun anotherNameGiven() {
         assertEquals("One for Bob, one for me.", twofer("Bob"))
     }
 
     @Test
-    @Ignore
```

Comment:
This should be ignored as in practice exericses, all tests are ignored except for the first.